python/vsi/tools/patch_wheel.py

```diff
@@ -1,77 +1,112 @@
 #!/usr/bin/env python
 
-import zipfile
-import pathlib
+from pathlib import Path
 import os
 import argparse
 import tempfile
+import subprocess
+import re
+from wheel.wheelfile import WHEEL_INFO_RE
 
 def get_parser():
   parser = argparse.ArgumentParser(
     description="Patch a wheel to change the package name or version in the "
-                "wheel metatdata. Note: This does not change any filenames or "
-                "version in the rest of the files (e.g. dist-info directory "
-                "name nor versions in .py files)")
+                "wheel metatdata. Note: This will also change any filenames or "
+                "versions in the rest of the file namess (e.g. dist-info "
+                "directory name) but will not change versions strings "
```

Comment:
```suggestion
                "wheel metatdata. Note: This may also change other filenames "
                "or versions in the rest of the files (e.g. dist-info directory "
                "name) but will not change versions strings "
```

Reply:
How can this change versions but not "version strings"

Reply:
As the next line say "hardcoded in .py files" suggest, if a py file says:

```
__version__ = "0.1.2"
```

I will not be updating that